stake-pool/js/src/utils/stake.ts

```diff
@@ -105,21 +125,26 @@ export async function prepareWithdrawAccounts(
   const withdrawFrom: WithdrawAccount[] = [];
   let remainingAmount = amount;
 
+  const fee = stakePool.stakeWithdrawalFee;
+  const inverseFee: Fee = {
+    numerator: fee.denominator.sub(fee.numerator),
+    denominator: fee.denominator,
+  };
+
   for (const type of ['preferred', 'active', 'transient', 'reserve']) {
```

Comment:
I tried removing transient from this list, but then the the error said:

```
Program log: Error withdrawing from reserve: validator stake accounts have lamports available, please use those first.
    Program log: Error: The lamports in the validator stake account is not equal to the minimum
```

Maybe the issue is way back in the `findStakeProgramAddress` function?